site/examples/progress_bars.js

```diff
@@ -5,6 +5,19 @@ const basic = `<div class='h12 bg-darken10 relative round-full'>
   <div class='absolute h12 bg-green-light round-full' style='width:50%;'></div>
 </div>`;
 
+const playback = `<div class='flex-parent flex-parent--row flex-parent--center-cross border border--gray-light round-t'>
+    <div class='flex-child py18 pr12 pl18 border-r border--gray-light bg-white cursor-pointer'>
+      <div class='triangle--r triangle color-green color-green'></div>
+    </div>
+    <div class='flex-child flex-child--grow p18 bg-white'>
+      <div class='relative w-full h12 bg-darken10 round-full'>
+        <div class='absolute top right bottom left m3 bg-green-light round-full' style='width:50%;'>
+          <span class='absolute top right w30 h6 bg-green round-full'></span>
```

Comment:
thanks @tristen. Is this what you meant with a range input?
<img width="1087" alt="screenshot 2017-02-15 15 40 37" src="https://cloud.githubusercontent.com/assets/7318064/22994491/203c701c-f395-11e6-980e-d9b2535f7e0b.png">